backend/test/features/activity-delete.feature

```diff
@@ -5,51 +5,42 @@ Feature: Delete an object
     Given our own server runs at "http://localhost:4123"
     And we have the following users in our database:
       | Slug          |
-      | bernd-das-brot|
-    And I send a POST request with the following activity to "/activitypub/users/bernd-das-brot/inbox":
+      | spongebob     |
+    And I send a POST request with the following activity to "/inbox":
```

Comment:
```suggestion
    And we have the following posts in our database:
```
Create a separate step definition for test setup. In this scenario I think only the `id` is relevant, no? Maybe content, too, but certainly not the entire activity json.